src/styles/main.scss

```diff
@@ -1,3 +1,76 @@
+$size : 100px;
+$weekDays :
+  "mon" 0,
+  "tue" ($size + 1),
+  "wed" (($size + 1) * 2),
+  "thu" (($size + 1) * 3),
+  "fri" (($size + 1) * 4),
+  "sat" (($size + 1) * 5),
+  "sun" (($size + 1) * 6);
+
+@mixin square($size) {
+  width: $size;
+  height: $size;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+}
```

Comment:
Use classes instead of tags for styling.
`body` dont have any `padding`.

Reply:
Thanks, I fixed